crates/api_models/src/payment_methods.rs

```diff
@@ -287,6 +287,11 @@ pub struct BankDebitTypes {
     pub eligible_connectors: Vec<String>,
 }
 
+#[derive(Debug, Clone, serde::Serialize, serde::Deserialize, ToSchema, PartialEq, Eq)]
```

Comment:
These changes are not required, Please remove this

Reply:
Removed